src/main/resources/application.yml

```diff
@@ -1,3 +1,16 @@
 logging:
   level:
     root: INFO
+
+server:
+  port: 8085
```

Comment:
The port has to be changed to 8085 also in the following files:-
ingress.yml
Dockerfile
deployment.yaml
service.yml
docker-compose.yml